frontend/src/metabase/query_builder/components/filters/modals/InlineOperatorSelector/InlineOperatorSelector.tsx

```diff
@@ -38,44 +39,49 @@ export function InlineOperatorSelector({
 
   return (
     <InlineOperatorContainer>
```

Comment:
these containers got shuffled around to enable us to 
- top align the field type icon and
- center align the field name
![Screen Shot 2022-08-01 at 6 29 56 PM](https://user-images.githubusercontent.com/30528226/182266868-9b0894eb-40ef-48df-ae43-e6f6a72e0521.png)